### ISSUE TITLE
add_device_type_components not working

### ISSUE THREAD

**Issue by PaulR282** [netbox-community/reports#67]
```
An exception occurred: TypeError: RearPortTemplate.instantiate() takes 1 positional argument but 2 were given

Traceback (most recent call last):
  File "/opt/netbox/netbox/extras/scripts.py", line 461, in _run_script
    script.output = script.run(data=data, commit=commit)
  File "/etc/netbox/scripts/add_device_type_components.py", line 44, in run
    items = [
  File "/etc/netbox/scripts/add_device_type_components.py", line 45, in <listcomp>
    x.instantiate(device)
TypeError: RearPortTemplate.instantiate() takes 1 positional argument but 2 were given

```

Getting this error on v3.2.7

**Comment by PaulR282** [comment]
> > File "/etc/netbox/scripts/add_device_type_components.py"
> 
> There's a bug in your custom script. FYI device component templates aren't meant to be instantiated directly.

Yes, I know, but it is a script from this repo.

**Comment by PaulR282** [comment]
fixed in #68
